wremnants/syst_tools.py

```diff
@@ -342,6 +347,16 @@ def add_QCDbkg_jetPt_hist(results, df, nominal_axes, nominal_cols, base_name="no
     qcdJetPt = dQCDbkGVar.HistoBoost(name, nominal_axes, [*nominal_cols, "nominal_weight"], storage=hist.storage.Double())
     results.append(qcdJetPt)
 
+def add_luminosity_unc_hists(results, df, args, axes, cols, addhelicity=False):
+    # TODO: implement for theory agnostic with addhelicity=True
+    if addhelicity:
```

Comment:
IS this called? Do you want to print out a warning rather than just silently doing nothing? If it's not called you can raise an NotImplementedError

Reply:
Right, I was going to implement it, so I didn't explicitly throw anything. Currently it is used for the W histmaker, but I still have to port it to Z ones.